boudicca.base/publisher-event-html/src/main/resources/templates/events/events.hbs

```diff
@@ -110,7 +120,11 @@
 <div style="text-align: center">
     <button id="loadMoreButton">Mehr laden</button>
 </div>
+<div id="resultNotFound" style="display: none">
+    <p>Keine Ereignisse gefunden für <span id="searchText"></span></p>
+    <p><a href="/" style="display: block">zurück zur Liste aller Ereignisse</a></p>
+</div>
 <div id="endOfResults" style="display: none">
-    <p>Du hast das Ende der Suchergebnisse erreicht.</p>
+    <p>Sie haben das Ende der Ergebnisliste erreicht.</p>
```

Comment:
only my opinion, but i like the "Du" more then "Sie", sounds friendlier to me

Reply:
done